video2dataset/downloader.py

```diff
@@ -9,17 +9,19 @@
 
 def handle_youtube(youtube_url):
     """returns file and destination name from youtube url."""
-    ydl_opts = {}
+    yt_dlp.utils.std_headers['Referer'] = "https://www.youtube.com/"
+    yt_dlp.utils.std_headers['User-Agent'] = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36'
+
+    ydl_opts = {
+        'quiet': True,
+        'format': 'bv*[height<=360][ext=mp4]+ba[ext=m4a]/b[height<=360]'
```

Comment:
can you tell me what this is looking for? like what does that string mean

Reply:
it selects best video in mp4 format but no better than 480p or the worst video if there is no video under 360p with the best audio in m4a format